src/System.Collections/tests/Generic/Dictionary/Dictionary.Tests.cs

```diff
@@ -225,6 +225,19 @@ public void CopyTo_NonContiguousDictionary(int count)
                 Assert.Equal(array[i++], obj.Key);
         }
 
+        [Fact]
+        public void Remove_NonExistentEntries_DoesNotThrow()
+        {
+            var dictionary = new Dictionary<string, string>();
+            dictionary.Add("a","b");
+            dictionary.Add("c","d");
+            foreach (var entry in dictionary)
+            {
+                if (dictionary.Remove(entry.Key + "Sibling"))
```

Comment:
I assume instead of "Sibling" it would just be `if (dictionary.Remove("nonexistent_key)`

Or is there some importance in it sorting between the two key values that I am missing? @benaadams